API/models/schemas/refs/base.json

```diff
@@ -22,6 +22,9 @@
           "type": "string",
           "pattern":"^\\w(\\w|\\-)*$"
         },
+        "hierarchyName": {
+          "type": "string"
+        },
```

Comment:
With this, you accept that the user send hierarchyName in a PUT/POST request, but hierarchyName will always be set by the API and not by the user, so it will be ignored.